lib/galaxy/visualization/genomes.py

```diff
@@ -26,7 +26,7 @@
 
 def decode_dbkey(dbkey):
     """ Decodes dbkey and returns tuple ( username, dbkey )"""
-    if isinstance(dbkey, str) and ':' in dbkey:
+    if isinstance(dbkey, basestring) and ':' in dbkey:
```

Comment:
Can you please use `six.string_types` instead for Python3 compatibility?

Reply:
Sure.